int32 command
float64 speed
